src/main/external-resources/documentation/introduction.html

```diff
@@ -152,19 +152,19 @@
 	<p>You can stream media files with quite old computers. If the
 		device does not recognize the video container format and UMS has to
 		remux you will need some additional CPU power. A 2 GHz CPU should be
-		sufficient. When transcoding a video on full HD resolution an up to
-		date dual core processor from Intel or AMD with at least 2,6 GHz is
+		sufficient. When transcoding a full HD resolution video, an up-to-date
+		dual core processor from Intel or AMD running at at least 2.6 GHz is
 		necessary.</p>
 
-	<p>It is also necessary to have enough network bandwidth available
-		when streaming HD videos. It is nearly impossible to stream a full HD
-		video via WLAN, even when restricting network bandwidth and setting
+	<p>You should also have enough network bandwidth available when streaming HD
+		videos. It is nearly impossible to stream a full HD video via
+		wireless LAN, even when restricting network bandwidth and setting
```

Comment:
@Nadahar I totally agree, and it's not @SubJunk that will say the opposite :wink: 

@dandv That documentation part should be modified.